packages/modules/create-imodel/package.json

```diff
@@ -19,8 +19,7 @@
     "clean": "rimraf cjs esm"
```

Comment:
now file upload is missing border, need this change
![image](https://github.com/iTwin/admin-components-react/assets/81580355/3f2d3717-d369-480f-a803-7a89fbf55a3c)

Reply:
Done